### DISCUSSION TITLE
this is a new error displayed i want to know the meaning of it

### DISCUSSION THREAD

**Discussion by Hamzastreamwide** [appium/appium#16475] [Q&A]
hi , does this mean the phone was locked ? and how i can prevent it if yes and thanks
org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: Unexpected data: {"Error":"PasswordProtected","Request":"StartService"}

**Comment by KazuCocoa** [comment]
Probably the device was locked, or similar locking thing in ios real devices. The message is by iOS, but they are basically undocumented. So we also do not know everything.
